app/app.py

```diff
@@ -1,9 +1,71 @@
-from flask import Flask, render_template
-app = Flask(__name__,
-            static_folder="./dist/static",
-            template_folder="./dist")
+from flask import Flask, Response, flash, render_template, request, redirect, session
+from .inmemory_image_store import InMemoryImageStore
 
+app_version = '0.0.1'
+
+app = Flask(__name__)
+app.secret_key = "f3cfe9ed8fae309f02079dbf"
```

Comment:
What does this do?

Reply:
The secret_key is needed in order to maintain sessions in Flask. We should make it secret in the future - https://flask.palletsprojects.com/en/0.12.x/quickstart/#sessions